landscape-report.md

```diff
@@ -0,0 +1,418 @@
+---
+title: "Sharing Government Software"
+---
+
+# Sharing Government Software
+
+
+### How Agencies Are Cooperatively Building Mission-Critical Software
+
+April 2021
+
+By Waldo Jaquith and Robin Carnahan
+
+State Software Collaborative
+
+Beeck Center for Social Impact + Innovation
+
+Georgetown University
+
+
+## About the Beeck Center for Social Impact + Innovation
+
+The Beeck Center is an experiential hub at Georgetown University that trains students and incubates scalable, leading edge ideas for social change. We believe impact at scale requires the courage to think and behave differently. Our work centers on investing in outcomes for individuals and society. We equip future global leaders with the mindset to promote outcome-driven solutions, using the tools of design, data, technology, and innovation. We convene actors across the public, private, and civic sectors to advance new tools, frameworks, and approaches necessary to achieve these outcomes.
+
+
+## About the State Software Collaborative
+
+The Beeck Center’s State Software Collaborative is bringing together U.S. software cooperatives to facilitate their development of shared software and services. Instead of 50 states buying 50 versions of near-identical, overpriced software, we’re facilitating the cooperative development of high-quality, fair-priced software to be shared among agencies.
+
+
+## About this Document
+
+This report was released April 2021 under a [Creative Commons Attribution-ShareAlike 4.0 license](https://creativecommons.org/licenses/by-sa/4.0/), and should be cited as: Waldo Jaquith and Carnahan, Robin. _Sharing Government Software: How Agencies Are Cooperatively Building Mission-Critical Software_. Beeck Center for Social Impact + Innovation, Georgetown University, 2021.
+
+
+
+
+## Table of Contents
```

Comment:
It'd be helpful to turn these list items into links to their relevant sections (which _should_ be doable with markdown links e.g. `[Executive Summary](#Executive-Summary)`)

Reply:
Done, will update